doc/users/whats_new/reproducible_ps_pdf.rst

```diff
@@ -2,5 +2,12 @@ Reproducible PS and PDF output
 ------------------------------
 
 The ``SOURCE_DATE_EPOCH`` environment variable can now be used to set
-the timestamps value in the PS and PDF outputs, which are then
-reproducible.
+the timestamps value in the PS and PDF outputs. See
+https://reproducible-builds.org/specs/source-date-epoch/
+
+Matplotlib does its best to make PS and PDF outputs reproducible, but
+be aware that some unreproducibility issues can arise if you use
+different versions of Matplotlib and the tools it relies on. Although
+standard plots has been checked to be reproducible, external tools can
```

Comment:
This is sort of nitpicking, but mathtext and the fonttype settings are internal to matplotlib. Here's my suggestion:

The reproducibility of the output from the PS and PDF backends has so far been tested using various plot elements but only default values of options such as `{ps,pdf}.fonttype`  that can affect the output at a low level, and not with the mathtext or usetex features. When matplotlib calls external tools (such as PS distillers or LaTeX) their versions need to be kept constant for reproducibility, and they may add sources of nondeterminism outside the control of matplotlib.